### PR TITLE
[SP-46] Added program which prompts user to guess the number

### PR DESCRIPTION
**Pull request by AndrewBuhanov**
Added program which prompts user to guess the ?-number in 10 attempts.

### CODE REVIEW

File: abuhanov/sp-46/src/random-number.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    int answer{};
+    do
+    {
+        std::cout << "Enter envisioned number " << std::endl;
+        std::cin >> answer;
+            if (n > answer) 
+            {
+                std::cout << "The number is bigger than you enter " << std::endl;
+            }
+            else if (n < answer)
```
Comment (matyushov) [inline abuhanov/sp-46/src/random-number.cpp]: упростить до if

File: abuhanov/sp-46/src/random-number.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    int n{};
+    std::srand (static_cast<unsigned int>(time(0)));
+    std::cout<<"Guess The Number :) " << std::endl;
+    n = (std::rand() % 100) + 1;
+    int answer{};
+    do
+    {
+        std::cout << "Enter envisioned number " << std::endl;
+        std::cin >> answer;
+            if (n > answer)
```
Comment (matyushov) [inline abuhanov/sp-46/src/random-number.cpp]: сдвинуть табиляцию блоков

File: abuhanov/sp-46/src/random-number.cpp (hunk 1 above)
Comment (AndrewBuhanov) [reply abuhanov/sp-46/src/random-number.cpp]: Done.

File: abuhanov/sp-46/src/random-number.cpp (hunk 2 above)
Comment (AndrewBuhanov) [reply abuhanov/sp-46/src/random-number.cpp]: Done.

**Review by matyushov — APPROVED** [review]

**Review by jackee67rus — APPROVED** [review]

**Review by jackee67rus — APPROVED** [review]

**Review by daltonya — APPROVED** [review]

**Review by jackee67rus — APPROVED** [review]

**Pull request closed without merging**